src/DataFixtures/AppFixtures.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace App\DataFixtures;
+
+use App\Entity\Game;
+use App\Entity\Player;
+use App\Entity\Ship;
+use App\Entity\Shoot;
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Persistence\ObjectManager;
+
+class AppFixtures extends Fixture
+{
+    public function load(ObjectManager $manager)
+    {
+        $game = new Game();
+        $game->setHash(('test'));
```

Comment:
Please give a more relevant value to your test fixtures. This way, if an error occured you won't be lost.

Eg: `$game->setHash('game_hash');`